### PR TITLE
[RFR] npm-shrinkwrap for production

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -38,7 +38,7 @@ install-aws: ## Install the aws cli
 
 install-prod: ## Install npm dependencies for the api, admin, and frontend apps in production environment
 	@echo "Installing Node dependencies"
-	@npm install
+	@npm install --production
```
Comment (djhi) [inline makefile]: Are you sure about this ? We use npm install for deployments and we need dev deps to build the static elements

File: package.json [hunk 2]
```diff
@@ -115,7 +115,7 @@
     "webpack-dev-server": "~1.14.1"
   },
   "scripts": {
-    "test": "make test"
+    "test": "npm ls"
```
Comment (djhi) [inline package.json]: no more tests ?

File: makefile (hunk 1 above)
Comment (Kmaschta) [inline makefile]: Why need we to build static elements in production ... ?

Comment (Kmaschta) [inline makefile]: Take a look on fabfile, I deploy static files locally.

Comment (djhi) [inline makefile]: ok, I thought it was done on the server

File: package.json (hunk 2 above)
Comment (Kmaschta) [inline package.json]: Just for tests purpose, I'll remove it.

**Pull request merged**